static/styles/right-sidebar.scss

```diff
@@ -142,14 +154,18 @@
     position: relative;
     top: 3px;
     border-radius: 4px;
-    margin-right: 15px;
+    margin-left: 5px;
 }
 
 .user_sidebar_entry .count {
-    margin-left: 2px;
+    margin-right: 0px;
```

Comment:
Is the `margin-right` property necessary? Doesn't look like there's a difference if I comment it out

Reply:
Nah, it was from before when I had not removed the margin-right from above, removed it.Thanks @synicalsyntax